src/Map/src/Bridge/Google/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/ux-map-google",
```

Comment:
Should be `symfony/ux-google-map` to be consistent with how we are naming bridges in Symfony.

Reply:
I'm working on it

Reply:
See #2031